requirements-minimal.txt

```diff
@@ -12,5 +12,5 @@ lxml>=4.6.5
 spikeextractors>=0.9.9
 spikeinterface>=0.93.0
 neo>=0.9.0
-roiextractors>=0.3.1
+roiextractors @ git+https://github.com/catalystneuro/roiextractors.git@875edc50e4cc392974c21e687199eb5349456af7
```

Comment:
point to master

Reply:
I really prefer to point to an actual commit (this is the most recent one to master) so that tests don't suddenly start failing on some random day as a consequence of some other changes.